src/features/fireWeather/components/graphs/WxDataGraph.tsx

```diff
@@ -74,7 +74,7 @@ const WxDataGraph = ({
   }
 
   const {
-    showObservations: showObservations,
+    showObservations,
```

Comment:
oopsie - did I not fix this as part of my PR?

Reply:
There were a bunch of warning because of redundant naming - so I just fixed it.

Reply:
Yeah you can see them on the terminal after starting `npm start`

Reply:
sorry guys - I 100% thought I'd fixed all of those and committed them before merging into main